### PR TITLE
[sidebar] Add tooltips to sidebar menu links

### PR DESCRIPTION
**Pull request by guilhermemntt**
Closes https://github.com/decred/decrediton/issues/2584.

### CODE REVIEW

**Comment by alexlyp** [comment]
Needs a rebase

**Comment by alexlyp** [comment]
Any update to tests @guilhermemntt ? still waiting on those fixes before we can merge this

**Comment by guilhermemntt** [comment]
@alexlyp I've fixed the unit tests. Please, do not merge this PR until https://github.com/decred/pi-ui/issues/295 is fixed. It's a bug in ``pi-ui`` tooltip component that seems to affect not only this specific PR, but all others related with ``pi-ui`` tooltip. Thanks!

**Comment by alexlyp** [comment]
OK thanks the update.  Waiting.

**Review by victorgcramos — CHANGES_REQUESTED** [review]
Hey @guilhermemntt thanks for the PR and sorry for the late feedback.
One thing is that Tooltips alignment should be improved when sidebar is collapsed on the left. Maybe place them on the right, what do you think?

Screen Shot 2020-10-20 at 12 09 06 PM

Left some inline comments as well!

**Comment by guilhermemntt** [comment]
Hey @victorgcramos, thanks for the review! I've just updated this PR with some fixes and improvements.

- The tooltip has been slightly moved to right, as you've sugested.

image

- I replaced the two ``map``s with a new functional component to increase the readability and modularity of the code.

@alexlyp @vctt94 sorry for the late reply. As the doubts regarding the tooltip visibility in small screens have been solved at https://github.com/decred/pi-ui/issues/295, I think this PR can be merged after the approve of the review.

File: app/components/SideBar/MenuLinks/MenuLink/MenuLink.jsx
Comment (guilhermemntt) [reply app/components/SideBar/MenuLinks/MenuLink/MenuLink.jsx]: Fixed.

File: app/components/SideBar/SideBar.module.css
Comment (guilhermemntt) [reply app/components/SideBar/SideBar.module.css:43]: By adding this ``CSS`` property at these components, we guarantee that their contents (tooltips here) won't be clipped by them. Otherwise, ``overflow: hidden`` would not be overrided and the we'd have this result:

image

File: app/components/SideBar/MenuLinks/MenuLinks.jsx
Comment (guilhermemntt) [reply app/components/SideBar/MenuLinks/MenuLinks.jsx]: Fixed.

File: app/components/SideBar/MenuLinks/MenuLinks.jsx
Comment (guilhermemntt) [reply app/components/SideBar/MenuLinks/MenuLinks.jsx]: Fixed.

File: app/components/SideBar/SideBar.module.css
Comment (victorgcramos) [reply app/components/SideBar/SideBar.module.css:43]: hmmmm nice, mate! Thanks for explaining :)

**Review by vctt94 — APPROVED** [review]
LGTM

**Pull request merged**
